README.md

```diff
@@ -7,9 +7,30 @@ repository-sync is a tool designed to keep two repositories—one private, one p
 
 ## Setup
 
-### Between two GitHub.com repositories
+### Deploy
 
-First, deploy this code to Heroku (or some other server you own).
+First, deploy this code to Heroku. Alternatively, use `docker-compose` (the app will listen on port `4567`):
+
+```bash
+docker-compose build
+docker-compose up
+```
+
+docker-compose expects a `.env` file with environment variables properly configured as below.
+
+Example:
+```
+GITHUB.ACME.COM_MACHINE_USER_TOKEN=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+GITHUB2.ACME.COM_MACHINE_USER_TOKEN=bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
```

Comment:
Hmm, what do these keys represent? I'm guessing they're supposed to correlate to `GHE1HOSTNAME_MACHINE_USER_TOKEN` / `GHE2HOSTNAME_MACHINE_USER_TOKEN`?

I'm a bit pressed for time at the moment, but, I think the README would be easier if this noted something like 

> docker-compose expects an `.env` file with environment variables properly configured. See the environment variables available defined below. For example:

Just because it seems like Docker can also do the DotCom-DotCom sync too, and I want users to know that it's not just limited to GHE!

Reply:
Yes, makes sense. I updated the sentence per your suggestion and also changed the example to be DocCom-GHE instead of GHE-GHE.